src/utils/testfiles/mockSearchResults.ts

```diff
@@ -11,7 +11,7 @@ export const mockSearchResults: SearchResponse2<RegistrationSearchItem, Registra
   totalHits: 50,
   hits: [
     mockRegistrationSearchItemWithMathJax,
-    mockRegistrationSearchItem,
+    { ...mockRegistrationSearchItem, identifier: '4327439' },
```

Comment:
Et lite triks for at testene skal kjøre nå som `mockMyRegistrations.ts` ikke kan brukes lenger. Skjørt, men vet ikke om det er verdt å buke mer tid på å rette det nå tbh 🤔